### PR TITLE
Add void method to SubscriptionService

### PR DESCRIPTION
**Pull request by swilczynski**

### CODE REVIEW

File: src/Services/SubscriptionService.php [hunk 1]
```diff
@@ -121,6 +121,14 @@ public function delete($subscriptionId)
         return $this->client()->delete('subscriptions/{subscriptionId}', ['subscriptionId' => $subscriptionId]);
     }
 
+    /**
+     * @param string $subscriptionId
```
Comment (slavcodev-rebilly) [inline src/Services/SubscriptionService.php:125]: What does this operation return?

File: src/Services/SubscriptionService.php [hunk 2]
```diff
@@ -121,6 +121,14 @@ public function delete($subscriptionId)
...
+     */
+    public function void($subscriptionId)
+    {
+        return $this->client()->post('subscriptions/{subscriptionId}/void', ['subscriptionId' => $subscriptionId]);
```
Comment (ashkarpetin) [inline src/Services/SubscriptionService.php:129]:
```suggestion
        return $this->client()->post([], 'subscriptions/{subscriptionId}/void', ['subscriptionId' => $subscriptionId]);
```

File: src/Services/SubscriptionService.php (hunk 1 above)
Comment (swilczynski) [reply src/Services/SubscriptionService.php:125]: Nothing, my mistake. Thanks for the tip

Comment (slavcodev-rebilly) [reply src/Services/SubscriptionService.php:125]: Alright, you've fixed another mistake, my question is about the return of `void` request though. According the API definitions returns "no content" (doubtful solution), but this method returns something.

Comment (swilczynski) [reply src/Services/SubscriptionService.php:125]: Added return type

**Review by creocoder — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Pull request merged**